python/cudnn/gemm/frost/kernel_templates/sm100_matmul_2ctamma_static.py

```
@@ -330,7 +332,34 @@ def _kernel(
                 for _ai in cutlass.range_constexpr(num_a_operands):
                     sA_stage = smem_a_list[_ai].subview(sA_elems * stage)
                     tma_a_desc = tma_a_descs[_ai]
-                    if cutlass.const_expr(multicast_a):
+                    if cutlass.const_expr(a_mcast_slices > 1):
```

Ruff (0.16.1)

[error] Undefined name `a_mcast_slices`

(F821)